BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Channel/TransmissionTest.cs

```diff
@@ -363,6 +366,50 @@ public async Task SendAsyncReturnsCorrectHttpResponseWrapperWithRetryHeaderWhenN
                 }
 
             }
+
+            [TestMethod]
+            public async Task SendAsyncLogsBreezeReponseTimeAndStatusCode()
+            {
+                var handler = new HandlerForFakeHttpClient
+                {
+                    InnerHandler = new HttpClientHandler(),
+                    OnSendAsync = (req, cancellationToken) =>
+                    {
+                        // VALIDATE
+                        Assert.AreEqual(testUri, req.RequestUri);
```

Comment:
these are unnecessary validations for this change

Reply:
Removed unnecessary checks.